datashuttle/utils/custom_exceptions.py

```diff
@@ -0,0 +1,6 @@
+class ConfigError(BaseException):
+    pass
+
+
+class NeuroBlueprintError(BaseException):
```

Comment:
```suggestion
class NeuroBlueprintError(Exception):
```
Someone might try to catch the exception using:
```
try:
    something
except Exception as e:
    cleanup
```
And not catch the exception if it inherits from `BaseException`.